rosa_tutorials/rosa-mobb-deploy-api-data-protection.adoc

```diff
@@ -0,0 +1,592 @@
+ "formatVersion": "1.1.0",
+ "phase": "Completed",
+ "progress": {
+   "itemsBackedUp": 58,
+   "totalItems": 58
+ },
+ "startTimestamp": "2022-09-07T22:20:22Z",
+ "version": 1
+}
+----
+
+. Delete the demo workload:
++
+[source,terminal]
+----
+$ oc delete ns hello-world
+----
+
+. Restore from the backup:
++
+[source,terminal]
+----
+$ cat << EOF | oc create -f -
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+ name: hello-world
+ namespace: openshift-adp
+spec:
+ backupName: hello-world
+EOF
+----
+
+. Wait for the Restore to finish:
++
+[source,terminal]
+----
+$ watch "oc -n openshift-adp get restore hello-world -o json | jq .status"
+----
++
+.Example output
+[source,JSON]
+----
+{
+ "completionTimestamp": "2022-09-07T22:25:47Z",
+ "phase": "Completed",
+ "progress": {
+   "itemsRestored": 38,
+   "totalItems": 38
+ },
+ "startTimestamp": "2022-09-07T22:25:28Z",
+ "warnings": 9
+}
+----
+
+. Check that the workload is restored:
++
+[source,terminal]
+----
+$ oc -n hello-world get pods
+----
++
+.Example output
+[source,terminal]
+----
+NAME                              READY   STATUS    RESTARTS   AGE
+hello-openshift-9f885f7c6-kdjpj   1/1     Running   0          90s
+----
++
+[source,terminal]
+----
+$ curl `oc get route/hello-openshift -n hello-world -o jsonpath='{.spec.host}'`
+----
++
+.Example output
+[source,terminal]
+----
+Hello OpenShift!
+----
+
+. For troubleshooting tips please refer to the OADP team's link:https://github.com/openshift/oadp-operator/blob/master/docs/TROUBLESHOOTING.md[troubleshooting documentation]
+
+. Additional sample applications can be found in the OADP team's link:https://github.com/openshift/oadp-operator/tree/master/tests/e2e/sample-applications[sample applications directory]
+
+[id="cleanup_{context}"]
+== Cleanup
+
+. Delete the workload:
++
+[source,terminal]
+----
+$ oc delete ns hello-world
+----
+
+. Delete the Data Protection Application:
++
+[source,terminal]
+----
+$ oc -n openshift-adp delete dpa ${CLUSTER_NAME}-dpa
+----
+
+. Delete the Cloud Storage:
++
+[source,terminal]
+----
+$ oc -n openshift-adp delete cloudstorage ${CLUSTER_NAME}-oadp
+----
++
+[WARNING]
+====
+If this command hangs, you might need to delete the finalizer:
+[source,terminal]
+----
+$ oc -n openshift-adp patch cloudstorage ${CLUSTER_NAME}-oadp -p '{"metadata":{"finalizers":null}}' --type=merge
+----
+====
+
+. Remove the Operator if it is no longer required:
++
+[source,terminal]
+----
+$ oc -n openshift-adp delete subscription oadp-operator
+----
+
+. Remove the namespace for the Operator:
++
+[source,terminal]
+----
+$ oc delete ns openshift-adp
+----
+
+. Remove the backup and restore resources from the cluster if they are no longer required:
++
+[source,terminal]
+----
+$ oc delete backup hello-world
+$ oc delete restore hello-world
+----
+
+. To delete the backup/restore and remote objects in s3:
++
+[source,terminal]
+----
+$ velero backup delete hello-world
+$ velero restore delete hello-world
+----
```

Comment:
Move this section above the "Delete the Data Protection Application" step above.